suite-native/module-dev-utils/src/screens/DevUtilsScreen.tsx

```diff
@@ -129,7 +129,14 @@ export const DevUtilsScreen = ({
                         {areLogsVisible && <Logs />}
                     </VStack>
                 </Box>
-            ) : null}
+            ) : (
+                <Card>
+                    <VStack spacing="medium">
+                        <CopyLogsButton />
```

Comment:
we don't want to display logs in production? These lines are IMHO missing:

```tsx
<CheckBox
   isChecked={areLogsVisible}
   onChange={() => setAreLogsVisible(!areLogsVisible)}
/>
{areLogsVisible && <Logs />}
```

Reply:
Won't be visible as discussed during standup.